examples/graphbolt/disk_based_feature/load_dataset.py

```diff
@@ -0,0 +1,62 @@
+import dgl.graphbolt as gb
```

Comment:
There is no need for this file. Let's just use the GraphBolt builtin datasets. For future reference, if you copy a file you should refer to where it is copied from.